unit-tests/run-unit-tests.py

```diff
@@ -421,6 +421,8 @@ def test_wrapper_( test, configuration=None, repetition=1, retry=0, sns=None ):
     if rslog:
         opts.add( '--rslog' )
     try:
+        if repetition > 0:
+            log.d("repeat #", repetition + 1)
```

Comment:
I think this is a retry, not a repetition? Oh wait... I think you want to check both retry and repetition here... but I think `configuration_str()` already does all this for you, why not use it?

Replace line 414 with:
```
if not log.is_debug_on() or repetition or retry:
```